spec/arguments_spec.lua

```diff
@@ -161,5 +161,15 @@ describe("tests related to positional arguments", function()
          }
          assert.has_error(function() parser:parse{} end, "missing argument 'foo1'")
       end)
+
+      it("handles invalid argument choices correctly", function()
+         local parse = Parser()
+         parse:argument "foo" {
+            choices = {"bar", "baz", "qu"}
+         }
+         assert.has_error(function()
+            parse:parse{"foo", "quu"}
```

Comment:
hrm, I was going to say that whitespace before the `{` is style for this codebase. but looks like it doesn't do that for one-liners?
I guess this is fine how you have it....